qa/L0_sequence_batcher/test.sh

```diff
@@ -229,28 +252,63 @@ fi
 
 for MODEL in $MODELS; do
   if [[ ! "$TEST_VALGRIND" -eq 1 ]]; then
-    cp -r $MODEL models1/. && \
+    if [[ "$MODEL" =~ .*"python".* ]]; then
+      generate_python_models "$MODEL" "models1"
       (cd models1/$(basename $MODEL) && \
-        sed -i "s/^max_batch_size:.*/max_batch_size: 4/" config.pbtxt && \
-        sed -i "s/kind: KIND_GPU/kind: KIND_GPU\\ncount: 1/" config.pbtxt && \
-        sed -i "s/kind: KIND_CPU/kind: KIND_CPU\\ncount: 1/" config.pbtxt)
-    cp -r $MODEL models2/. && \
+            sed -i "s/^max_batch_size:.*/max_batch_size: 4/" config.pbtxt && \
+            sed -i "s/kind: KIND_GPU/kind: KIND_GPU\\ncount: 1/" config.pbtxt && \
+            sed -i "s/kind: KIND_CPU/kind: KIND_CPU\\ncount: 1/" config.pbtxt)
+    else
+      cp -r $MODEL models1/. && \
+        (cd models1/$(basename $MODEL) && \
+          sed -i "s/^max_batch_size:.*/max_batch_size: 4/" config.pbtxt && \
+          sed -i "s/kind: KIND_GPU/kind: KIND_GPU\\ncount: 1/" config.pbtxt && \
+          sed -i "s/kind: KIND_CPU/kind: KIND_CPU\\ncount: 1/" config.pbtxt)
```

Comment:
Exclude these lines from the if-else condition.

Reply:
Why exclude? Are you suggesting to merge it with the same code in the preceding if condition? I had previously done this but found that it decreased readability.

Reply:
Exactly. This is to avoid code duplication. If we decide to change something, we would have to change it in two places.

Reply:
Clauses merged